test/unit/aggregators/log-aggregator.test.js

```diff
@@ -128,3 +138,58 @@ test('Log Aggregator', (t) => {
     t.end()
   })
 })
+
+test('big red button', (t) => {
+  t.autoend()
+
+  let agent
+
+  t.beforeEach(() => {
+    // setup agent
+    agent = helper.instrumentMockedAgent()
+  })
+
+  t.afterEach(() => {
+    agent && helper.unloadAgent(agent)
+  })
+
+  t.test('should show logs if the config for it is enabled', (t) => {
+    t.plan(2)
+
+    agent.config.onConnect({
+      event_harvest_config: {
+        report_period_ms: 60,
+        harvest_limits: {
+          log_event_data: 42
+        }
+      }
+    })
+    agent.onConnect(false, () => {
+      agent.logs.add('hello')
+      agent.logs.add('world')
```

Comment:
This fails the unit tests because of this line, which attempts to parse 'hello' and 'world' as JSON expressions: https://github.com/newrelic/node-newrelic/blame/ce80fa42d1d2065dfa00baeab147af710afe4e45/lib/aggregators/log-aggregator.js#L46

Should the log aggregator assume all inputs are objects or JSON-parseable expressions? If so, our tests should supply the aggregator with appropriate values.

Reply:
Ah, oops, I think I rebased and didn't redo the tests after the rebase. Should be fixed now.